templates/default/datadog.conf.erb

```diff
@@ -72,6 +72,8 @@ max_traces_per_second: <%= node['datadog']['max_traces_per_second'] %>
 [trace.receiver]
 receiver_port: <%= node['datadog']['receiver_port'] %>
 connection_limit: <%= node['datadog']['connection_limit'] %>
+
+[Main]
```

Comment:
Actually this works, but I think it would be more human-readable to have the whole `Main` section in one block.

Could you move the whole trace-agent-related part of the config at the end of the file so that the `Main` section is not split?